bigchaindb_driver/driver.py

```diff
@@ -420,7 +420,7 @@ def get(self, *, txid, status=None, headers=None):
             headers (dict): Optional headers to pass to the request.
 
         Returns:
-            :obj:`list` of :obj:`str`: List of block ids.
+            :obj:`list` of :obj:`int`: List of block heights.
```

Comment:
Since we don't store invalid blocks anymore, the transaction can only be in one block. Because of that, `status` is also not used anymore. We can keep `status`, ignore it, and display a warning saying it will be deprecated soon.